### DISCUSSION TITLE
仓库的宣传

### DISCUSSION THREAD

**Discussion by aqni** [Tr0py/NKU-thesis-template-2020#29] [General]
续 https://github.com/Tr0py/NKU-thesis-template-2020/issues/27#issue-1710795169

直接联系教务处似乎不可行，当前模板使用者基数不够大，无法引起官方重视。

我认为可行的几个方式：

- 联系在南开大学中有影响力的私人微信公众号，投稿宣传文章
- 在知乎、B站等流量较大社交媒体投稿。
- 在[NKUCS](https://nkucs.icu/#/)之类的信息分享平台投稿。
- 联系物理学院和数学学院（这两个需要latex使用者较多）的模板维护者，将力量集中起来。
- 与硕博latex模板互动，添加友链。

**Comment by Tr0py** [comment]
好哇！good points！

话说你说的“物理学院和数学学院的模板维护者”是谁呀？他们有自己的仓库吗？

**Reply by aqni** [reply]
物理学院和数学院的模板，似乎已经有一段时间不维护了，但是我觉得后续应该会有人会接手。
- 物理学院学长模板主页 https://article.benhaotang.cn/pc/26.html
- 数学院 https://github.com/skyline77/NKU-biyelunwen-2021

**Reply by aqni** [reply]
如果是在校生我可以尝试通过飞书联系一下

**Reply by benhaotang** [reply]
我的在这里呢https://github.com/benhaotang/NKThesis ，我这是当时因为理论物理的老师都受不了word了然后说想要弄一个更新版的latex模板才做的，但是鉴于是改的旧代码，似乎03年就开始用了，实际上越改越变成屎山了
目前物理学院和数学学院分发模板的方式是最后一年由本学院的教务老师发到QQ或微信群中，所以这也导致一个问题，就是他们经常传旧版本导致我总是收到一些已经修复的问题的提问，而且不同的老师审核模板会给出相反的意见，这也是为啥我的bibtex样式给了倆，其他一般都是给我提我尽量帮他们改。我觉得最好的办法就是联系对应学院的教务老师把你的模板的链接给他们，如果那些行政的老师看过没有问题后，自然就传播开了，因为我觉得这样的分发办法情况下大家不会上网搜其他模板的。另外也可以办一些讲座，我的repo里就有当时workshop的beamer。
